src/main/java/core/basesyntax/strategy/ReturnOperationHandler.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax.strategy;
+
+import core.basesyntax.dao.FruitStorageDao;
+import core.basesyntax.model.FruitTransaction;
+
+public class ReturnOperationHandler implements OperationHandler {
+    @Override
+    public void updateFruitStorage(FruitTransaction fruitTransaction,
+                                   FruitStorageDao fruitStorageDao) {
+        fruitStorageDao.increaseQuantity(fruitTransaction);
+    }
```

Comment:
According to the checklist, the `ReturnOperationHandler` class should not begin with an empty line. Please remove the empty line after the class declaration to follow the Java code style guidelines.